kubectl-plugin/pkg/util/kubeconfig.go

```diff
@@ -0,0 +1,47 @@
+package util
+
+import (
+	"path/filepath"
+	"testing"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
+)
+
+// HasKubectlContext checks if the kubeconfig has a current context or if the --context switch is set.
+func HasKubectlContext(config api.Config, configFlags *genericclioptions.ConfigFlags) bool {
+	return config.CurrentContext != "" || (configFlags.Context != nil && *configFlags.Context != "")
+}
+
+// CreateTempKubeConfigFile creates a temporary kubeconfig file with the given current context.
+// This function should only be used in tests.
+func CreateTempKubeConfigFile(t *testing.T, currentContext string) (string, error) {
```

Comment:
nit (not blocking): Test-only functions usually live in _test.go files

Reply:
gotcha, should I move this into `kubeconfig_test.go` in the same dir?